server/app/import.go

```diff
@@ -239,6 +247,27 @@ func (a *App) ImportBoardJSONL(r io.Reader, opt model.ImportArchiveOptions) (str
 		}
 	}
 
+	if len(boardMembers) > 0 {
+		for _, board := range boardsAndBlocks.Boards {
```

Comment:
Shouldn't there be a different set of boardMembers for each board. Is there only one board in this range?

Reply:
@sbishel I think now we can only export one board at a time instead of the whole workspace in the workspace era. So I believe an import will consist of only one board. However, I'm not sure if we're still supporting import of the original workspace format imports. If we are, that can definitely contain multiple boards, but then it won't have board members.